ariadne/cardano/test/backend/Test/Spec/Keystore.hs

```diff
@@ -70,11 +66,11 @@ spec =
                     (fmap hash mbKey) `shouldBe` (Just (hash esk))
 
         prop "Inserts are persisted after releasing the keystore" $ monadicIO $ do
-            keystorePath <- getKeystorePath
+            keystorePath <- getTempKeystorePath
             (STB wid, STB esk) <- pick genKeypair
             run $ do
                 nukeKeystore keystorePath
-                Keystore.bracketKeystore KeepKeystoreIfEmpty keystorePath $ \keystore1 ->
+                Keystore.bracketKeystore KeepKeystoreIfEmpty keystorePath $ \keystore1 -> do
```

Comment:
Redundant change.

Reply:
I think it's OK as it makes the code more uniform.